deployment/v3/docs/external services configuration.md

```diff
@@ -0,0 +1,82 @@
+#  External Services Configuration
+
+* This document describes how to configure external services like Minio,Keycloak, and Postgres
```

Comment:
This document describes how to configure external services like MinIO, Keycloak, and Postgres. (* not required at the start)